06-lifecycle_of_a_Patch.Rmd

```diff
@@ -22,6 +22,71 @@ To submit a patch, you need:
 
 Refer to the guidelines given [here](https://www.r-project.org/bugs.html#how-to-submit-patches) for the process to prepare a patch against R-devel.
```

Comment:
I don't think this sentence is needed any more, now you are giving instructions tailored to how we have suggested people set things up.